app/assets/javascripts/discourse/app/widgets/header.js

```diff
@@ -151,6 +161,10 @@ createWidget("header-notifications", {
   },
 
   _addAvatarHighlight(contents) {
+    if (this.siteSettings.enable_onboarding_popups) {
```

Comment:
This is odd. Above, `_addAvatarHighlight` is only called when `enable_onboarding_popups` is `true`, but then here we exit early when that is the case.

Reply:
`_addAvatarHighlight` is only called when `enable_onboarding_popups` is `false`. The call comes is in the `else` branch.

Reply:
This was just a sanity check, but I don't think it is worth having it at all.